cmake/FindLibTrixi.cmake

```diff
@@ -0,0 +1,34 @@
+#
+# Stop if already found
+#
+if ( LIBTRIXI_FOUND )
+    return()
+endif()
+
+
+
+#
+# Look for include, libs, executables
+#
+find_path ( LIBTRIXI_INCLUDE_DIR trixi.h PATHS ${LIBTRIXI_PREFIX}/include )
+find_file ( LIBTRIXI_VERSION_FILE LIBTRIXI_VERSION PATHS ${LIBTRIXI_PREFIX} )
```

Comment:
I think `find_file` is not recursive, so you need to set the path to something like `${LIBTRIXI_PREFIX}/share/julia`